chrome/background.js

```diff
@@ -85,6 +85,14 @@ function onMessage(request, sender, sendResponse) {
   if (request.action == "dismissOTP" && sender.tab.id in tabInfos) {
     delete tabInfos[sender.tab.id];
   }
+
+  // allows the local communication to request settings. Returns an
+  // object that has current settings. Update this as new settings
+  // are added (or old ones removed)
+  if (request.action == "getSettings") {
+    const use_fuzzy_search = localStorage.getItem("use_fuzzy_search") == "true";
```

Comment:
Since I see you agreed to make fuzzy search the default, change here too: `!= "false"`